### PR TITLE
rootfs-configs.yaml: Include armel,armhf architecture

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Signed-off-by: Lakshmipathi <lakshmipathi.ganapathi@collabora.com>

### CODE REVIEW

**Review by gctucker — COMMENTED** [review]
The change looks fine, but the commit message is not descriptive enough.

The summary should mention buster-ltp, and having a more detailed commit message to explain why we're doing this (or why armel and armhf were not built before) would be helpful.

File: rootfs-configs.yaml [hunk 1]
```diff
@@ -143,4 +143,6 @@ rootfs_configs:
     arch_list:
       - amd64
       - arm64
+      - armel
```
Comment (gctucker) [inline rootfs-configs.yaml:146]: Actually, `armel` is not really needed.  We can run LTP with `armhf` on all the 32-bit arm platforms we have.  That's what the other Buster rootfs images do.

Comment (Lakshmipathi) [reply rootfs-configs.yaml]: okay  updated the patch.

**Comment by Lakshmipathi** [comment]
Now added more details to commit message.

**Comment by gctucker** [comment]
Thanks @Lakshmipathi. Have you built the `armhf` variant on staging already?

**Comment by gctucker** [comment]
I see this was tested as described on #520, here's a sample job: https://lava.collabora.co.uk/results/2741325/0_ltp

**Pull request merged**